news/vulnerabilities.xml

```diff
@@ -1594,6 +1594,53 @@ issue.
     <advisory url="/news/secadv/20170126.txt"/>
     <reported source="OSS-Fuzz project" />
   </issue>
+
+  <issue public="20160824">
+    <impact severity="Low"/>
+    <cve name="2016-2183"/>
+    <affects base="1.0.2" version="1.0.2"/>
+    <affects base="1.0.2" version="1.0.2a"/>
+    <affects base="1.0.2" version="1.0.2b"/>
+    <affects base="1.0.2" version="1.0.2c"/>
+    <affects base="1.0.2" version="1.0.2d"/>
+    <affects base="1.0.2" version="1.0.2e"/>
+    <affects base="1.0.2" version="1.0.2f"/>
+    <affects base="1.0.2" version="1.0.2g"/>
+    <affects base="1.0.2" version="1.0.2h"/>
+    <fixed base="1.0.2" version="1.0.2i" date="20160922"/>    
+    <problemtype>cryptographic weakness</problemtype>
+    <title>SWEET32 attack on Triple-DES</title>
+    <description>
+      Because DES (and triple-DES) has only a 64-bit block size,
+      birthday attacks are a real concern. With the ability to run
+      Javascript in a browser, it is possible to send enough traffic
```

Comment:
Is this attack predicated on Javascript being available or does that make the attack easier?